### PR TITLE
Added Auth with Toast

### PR DESCRIPTION
**Pull request by VLeads**
Added Authentication
Added Toast messages with auth feature.

Created some extra files related to it.

Commit to check:  Added Auth with toast messages

Demo: https://deploy-preview-5--rapidstore-dev.netlify.app/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import logo from "./logo.png";
 
 import { Header, Footer } from "./components";
 
-import { Homepage, Productpage } from "./pages";
+import { Homepage, Productpage, Wishlist } from "./pages";
```
Comment (Shubhamkr2610) [inline src/App.js]: Appreciation: its great to see  that you are using absolute import

File: src/pages/wishlist/Wishlist.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            <p class="card-price">
+              <span class="price">₹96,990 </span>
+              <strike>₹1,36,990</strike>
+            </p>
+          </div>
+          <div class="card-footer">
+            <button class="btn btn-primary">Add to Cart</button>
+            <button class="btn-card heart-select">
+              <i class="fa fa-heart"></i>
+            </button>
```
Comment (Shubhamkr2610) [inline src/pages/wishlist/Wishlist.jsx:29]: replace the class with className for better practice

**Review by Shubhamkr2610 — APPROVED** [review]
Everything looks good to me . Your is clean and readable , easy to understand . gave some suggestions too. You Can merge it after making the changes

File: src/components/header/Header.jsx [hunk 3]
```diff
@@ -39,17 +62,23 @@ export function Header() {
...
+            <Link to="/wishlist">
               <span className="badge-icon">
                 <span className="badge red">20+</span>
                 <i className="far fa-heart"></i>
               </span>
-            </a>
+            </Link>
           </li>
           <li>
             <a href="./components/cart/cart.html">
```
Comment (ApurvaSawant11) [inline src/components/header/Header.jsx:84]: Use Link tag instead of anchor tags

**Review by ApurvaSawant11 — APPROVED** [review]
The code looks good and the UI is also great.

Just check if login and signup are working or not for the **deployed site**. It was not working for me.

**Comment by VLeads** [comment]
> The code looks good and the UI is also great.
> 
> Just check if login and signup are working or not for the **deployed site**. It was not working for me.

Thanks for review Apurva. Auth was not working on deploy, because environment variable was not there in vercel. I have added that. If you can, you can try it now, it will work fine !!

File: src/components/header/Header.jsx (hunk 3 above)
Comment (VLeads) [reply src/components/header/Header.jsx:84]: I am working on cart and wishlist. I'll  update them in new PR

File: src/pages/wishlist/Wishlist.jsx (hunk 2 above)
Comment (VLeads) [reply src/pages/wishlist/Wishlist.jsx:29]: I'll update them in new PR.

**Comment by VLeads** [comment]
> Everything looks good to me . Your is clean and readable , easy to understand . gave some suggestions too. You Can merge it after making the changes

Thanks for review Shubham and for appreciation too. I'll consider those changes in new PR.

**Pull request merged**